opal/mca/btl/tcp/btl_tcp.c

```diff
@@ -531,3 +533,64 @@ void mca_btl_tcp_dump(struct mca_btl_base_module_t* base_btl,
     }
 #endif /* OPAL_ENABLE_DEBUG && WANT_PEER_DUMP */
 }
+
+
+/*
+ * A blocking recv on a non-blocking socket. Used to receive the small
+ * amount of connection information that identifies the endpoints
+ * endpoint.
+ */
+
```

Comment:
Just wondering since EINPROGRESS is not handled in the error cases of mca_btl_tcp_recv_blocking, we will loop through till we get value. so I hope EINPROGRESS is also handled indirectly here. Correct me if i am wrong.  

https://github.com/mohanasudhan/ompi/blob/master/opal/mca/btl/tcp/btl_tcp.c#L544

Definetly comments has to be improved here. Let me know if i missing anything. Thanks for comments.